apps/vaporgui/CopyRegionAnnotationWidget.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "CopyRegionWidget.h"
+
+namespace VAPoR {
+class ControlExec;
+class RenderParams;
+class ParamsMgr;
+class DataMgr;
+}    // namespace VAPoR
+
+class CopyRegionAnnotationWidget : public CopyRegionWidget {
+    Q_OBJECT
+
+public:
+    CopyRegionAnnotationWidget(VAPoR::ControlExec *ce);
+
+    QString name() const { return "CopyRegionAnnotationWidget"; }
+    QString includeFile() const { return "CopyRegionAnnotationWidget.h"; }
```

Comment:
What are these for?

Reply:
These are legacy functions that should have been removed.  Fixed.